packages/devtools/lib/src/debugger/debugger.dart

```diff
@@ -35,7 +35,12 @@ import '../utils.dart';
 class DebuggerScreen extends Screen {
   DebuggerScreen()
       : debuggerState = DebuggerState(),
-        super(name: 'Debugger', id: 'debugger', iconClass: 'octicon-bug') {
+        super(
+          name: 'Debugger',
+          id: 'debugger',
+          iconClass: 'octicon-bug',
+          disabled: shouldHideTab('debugger'),
```

Comment:
instead of specifying disabled here, have the main Screen constructor take 
disabled as an argument.

Reply:
Done!